frontends/qt/setup.nsi

```diff
@@ -103,6 +104,9 @@ Section -Main SEC0000
     #File /r /x Makefile* @abs_top_srcdir@/doc\*.*
     SetOutPath $INSTDIR
     WriteRegStr HKCU "${REGKEY}\Components" Main 1
+
+    # Create a shortcut on the desktop
+    CreateShortCut "$DESKTOP\${APP_NAME}.lnk" "$INSTDIR\${APP_EXE}" "" "" 0
```

Comment:
Other lines use `$(^Name)`, presumably to reference the Name "BitBoxApp" as defined in the first line, so I think you can avoid defining another APP_NAME variable and use that instead.